Resources/Prototypes/Entities/Objects/Tools/jaws_of_life.yml

```diff
@@ -18,15 +18,15 @@
     slots:
       - Belt
   - type: ToolTileCompatible
+    delay: 0.5
```

Comment:
Idk if it's too fast. Let me know if the value needs to be changed or should I keep it default.